tools/kind/Dockerfile.webhook-proxy

```diff
@@ -0,0 +1,8 @@
+FROM registry.access.redhat.com/ubi9/nginx-124
```

Comment:
Could you add a comment explaining why this here? It might be confusing to new developers.